Assets/GothicVR/Scripts/Npc/Actions/AnimationActions/GoToWp.cs

```diff
@@ -21,13 +22,20 @@ public override void Start()
              * 1. AI_StartState() can get called multiple times until it won't share the WP. (e.g. ZS_SLEEP -> ZS_StandAround())
              * 2. Happens (e.g.) during spawning. As we spawn NPCs onto their current WayPoints, they don't need to walk there from entrance of OC.
              */
-            if (destination == "" || Props.CurrentWayPoint.Name == destination)
+            if (Props.CurrentWayPoint != null && (destination == "" || Props.CurrentWayPoint.Name == destination))
```

Comment:
It seems like something's set wrong before this AIHandler is called.
OR: Did you forget to use the WP which is set for this call?